start-page/beta/index.html

```diff
@@ -2,39 +2,169 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "_base-resp.html" %}
+{% extends "_new-base-resp.html" %}
 
 {% block channel %}beta{% endblock %}
-{% block title %}{{ _('Welcome to <span>Thunderbird Beta</span>') }}{% endblock %}
+{% block title %}{{ _('Welcome to <div class="tb-text">Thunderbird <small class="beta-text">Beta</small></div>') }}{% endblock %}
 
 {% block main %}
       <header>
-        <h1>{{ self.title() }}</h1>
-        <p>{{ _('You are using the beta version of Thunderbird, which means you get to experience improvements and fixes before the rest of our users! But with great power comes great responsibility, we hope you’ll take the time to report bugs and provide feedback on your experience. Thank you for helping to make the next Thunderbird release as awesome as possible. <a class="more" href="%(learn)s">Learn more about the Beta.</a>')|format(learn='https://support.mozilla.org/kb/thunderbird-beta') }}</p>
-        <p><b>{{ _('WARNING: Add-ons in Thunderbird Betas can be fragile, meaning they may not work properly in your current beta, may break in a future Thunderbird Beta, and you may have installation problems.  Anyone using or thinking of using add-ons in beta versions should read our <a href="%(addons)s">add-ons KB article</a>, especially if you are currently using Thunderbird Beta version 60 or older.')|format(addons='https://support.mozilla.org/kb/thunderbird-beta#w_add-ons')}}</b></p>
+        <div class="header-top">
+          <div class="title">
+            <div class="logo-img"></div>
+            <h1 class="title-text">{{ self.title() }}</h1>
+          </div>
+
+          <div class="donate">
+            Support us!
+            <a href="https://give.thunderbird.net/?utm_source=start_page_tb_beta&utm_medium=referral&utm_content=paragraph_text" class="donate-btn">
```

Comment:
This link must use the `donate_url` function as in the old start page: https://github.com/thundernest/thunderbird-website/blob/master/start-page/beta/index.html#L28

Reply:
Is there a way to pull in the heart svg with the link formatted like this?

Reply:
Sure, donate_url only generates a URL, you can change classes or anything on the tag itself.
```
{{ '<a href="%(donate)s" class="donate-btn">'|format(donate=donate_url('paragraph_text', 'start_page_tb_beta')) }}
{{ svg('heart') }}
{{ _('Donate') }}
</a>
```